client/reader/discover/discover-stream.js

```diff
@@ -23,6 +23,33 @@ import {
 	FIRST_POSTS_TAB,
 } from './helper';
 
+export const DiscoverHeader = ( props ) => {
+	const translate = useTranslate();
+
+	const { selectedTab } = props;
+	const tabTitle = getSelectedTabTitle( selectedTab );
+	let subHeaderText = translate( 'Explore %s blogs that inspire, educate, and entertain.', {
+		args: [ tabTitle ],
+		comment: '%s is the type of blog being explored e.g. food, art, technology etc.',
+	} );
+	if ( selectedTab === FIRST_POSTS_TAB ) {
+		subHeaderText = translate(
+			'Fresh voices, fresh views. Explore first-time posts from new bloggers.'
+		);
+	}
+
+	return (
+		<NavigationHeader
+			navigationItems={ [] }
+			title={ translate( 'Discover' ) }
+			subtitle={ subHeaderText }
+			className={ clsx( 'discover-stream-header', {
+				'reader-dual-column': Number.isInteger( props.width ) && props.width > WIDE_DISPLAY_CUTOFF,
```

Comment:
Is the `Number.isInteger` check really needed? It will work just fine if `props.width` is `undefined` or `null`. 

```suggestion
				'reader-dual-column': Number.isInteger( props.width ) && props.width > WIDE_DISPLAY_CUTOFF,
```

Reply:
Thank you! I've removed the check.